opus/application/static_media/css/opus.css

```diff
@@ -2281,12 +2288,14 @@ to the left pane when screen is getting too wide */
         flex-wrap: wrap-reverse;
     }
 
-    /* Set detail image size when it's above the left pane */
+    /* Set detail image size when it's above the left pane, use !important to override
+    the styling from flexslider */
     .op-detail-img img {
-        height: 150px;
+        height: 150px !important;
     }
 
-    /* Reduce the top space above the detail image when screen */
+    /* Reduce the top space above the detail image when screen, use !important to override
```

Comment:
I'm not sure what this comment was supposed to mean originally - "when screen" doesn't make any sense. Can you fix it?